templates/careers/job-detail.html

```diff
@@ -89,7 +89,12 @@ <h3>Apply for this role</h3>
           <button type="button" class="js-locate-me">Locate me</button>
           <span class="js-locate-error u-hide" style="color: #c7162b">Location unavailable</span>
           <hr style="margin: 1rem 0;" />
-          <button type="submit" class="p-button--positive u-no-margin--bottom">Submit Application</button>
+          <button type="submit" class="p-button--positive u-no-margin--bottom" onsubmit="dataLayer.push({
```

Comment:
The `onsubmit` event is triggered from the form element, not the button element.

Reply:
Moved to an `onclick`